Aspire Budgeting/Views/TabBarView/TabBarView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  TabBarView.swift
+//  Aspire Budgeting
+//
+
+import SwiftUI
+
+struct TabBarView: View {
+  
+  private let cornerRadius: CGFloat = 16
+  private let height: CGFloat = 95
+  private let shadowRadius: CGFloat = 16
+  
+  private let prominentItemWidth: CGFloat = 70
+  
+  private var prominentItemTopPadding: CGFloat {
+    return -prominentItemWidth
+  }
+  
+  let tabBarItems: [TabBarItem]
+  let prominentItemImageName: String
+  @Binding var selectedTab: Int
+  let prominentItemAction: () -> Void
+  
+//  @State var selectedIndex = 0
+  
+  var body: some View {
+    ZStack {
+      containerBox
+      prominentItemView
+      tabBarItemsView
+    }
+  }
+}
+
+extension TabBarView {
+  private var containerBox: some View {
+    Rectangle()
+      .fill(Color.tabBarColor)
+      .cornerRadius(cornerRadius)
+      .frame(height: height)
+      .shadow(radius: shadowRadius)
+  }
+  
+  private var prominentItemView: some View {
+    ProminentTabBarItemView(systemImageName: prominentItemImageName) {
+      self.prominentItemAction()
+    }
+    
+    .padding(.top, prominentItemTopPadding)
+  }
+  
+  private func tabBarItemWidth(from proxy: GeometryProxy) -> CGFloat {
+    return (proxy.frame(in: .global).width - prominentItemWidth) / 4
+  }
+  
+  private var tabBarItemsView: some View {
+    GeometryReader { geo in
+      HStack {
+        ForEach(0..<2) { idx in
+          TabBarItemView(tabBarItem: self.tabBarItems[idx],
+                         selectedIndex: self.selectedTab,
+                         tabBarIndex: idx,
+                         defaultColor: .tabBarItemDefaultTintColor,
+                         selectedColor: .tabBarItemSelectedTintColor,
+                         font: .nunitoBold(size: 14))
+            .frame(width: self.tabBarItemWidth(from: geo))
+            .onTapGesture {
+              self.selectedTab = idx
+          }
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/aspirebudgetingmobile/aspirebudgeting_ios/blob/845606672c945b4a26becac88e482112dc55b51f/Aspire Budgeting/Views/TabBarView/TabBarView.swift#L70"></span>Closure end should have the same indentation as the line that started it. Expected 12, got 10.<br />
<code>closure_end_indentation</code> <code>TabBarView.swift:70</code></td>
    </tr>
  </tbody>
</table>